bitrepository-webclient/src/main/webapp/integrity-service.html

```diff
@@ -203,71 +202,92 @@ <h3 id="modalPagerLabel">Modal header</h3>
         });
     }
 
-    function getPagingLimit(id, member) {
-        var myID = id;
-        var myMember = member;
-        return function() {
-            return pillars[myID][myMember];
+    function startWorkflow() {
+        let ID = $("#workflowSelector option:selected").val();
+        let url = integrityServiceUrl + '/integrity/IntegrityService/startWorkflow/';
+        $('#formStatus').load(url, {
+            workflowID: ID,
+            collectionID: getCollectionID()
+        }).show().fadeOut({duration: 5000});
+    }
+
+    function getMaxPages(pillarID, member) {
+        return function () {
+            return pillars[pillarID][member];
         }
     }
```

Comment:
Can't `getMaxPages` just return `pillars[pillarID][member]` directly instead of returning it as a function that is called later on?

Reply:
This has been updated with both a more meaningful name and a comment explaining why an anonymous function is returned.